Update kyma-operator readme (#8703)
kyma-project/kyma: components/kyma-operator/README.md

### Custom resource file

The [Installation custom resource file](https://kyma-project.io/docs/root/kyma/#custom-resource-installation) provides the basic information for Kyma installation.

> ### Configuration

Reviewer: ```suggestion
## Configuration
```

> Operator tracks the Installation CR and waits for overrides in a given namespace. You can configure them using those

Reviewer: ```suggestion
Kyma Operator tracks the Installation CR and waits for overrides in a given Namespace. You can configure the overrides using these
```

> environmental variables. 

Reviewer: ```suggestion
environmental variables:
```

| Name                	| Default                               	| Description                                            	|
|---------------------	|---------------------------------------	|--------------------------------------------------------	|

> | INST_NAMESPACE      	| `default`                             	| Namespace in which the Installation is located         	|

Reviewer: ```suggestion
| **INST_NAMESPACE**      	| `default`                             	| Namespace in which the Installation CR is located         	|
```

> | INST_RESOURCE       	| `kyma-installation`                   	| Name of the Installation resource                      	|

Reviewer: ```suggestion
| **INST_RESOURCE**       	| `kyma-installation`                   	| Name of the Installation custom resource                      	|
```

> | INST_FINALIZER      	| `finalizer.installer.kyma-project.io` 	| Name of the finalizer                                  	|

Reviewer: ```suggestion
| **INST_FINALIZER**      	| `finalizer.installer.kyma-project.io` 	| Name of the finalizer                                  	|
```

> | OVERRIDES_NAMESPACE 	| `kyma-installer`                      	| Namespace in which the Installer Overrides are located 	|

Reviewer: ```suggestion
| **OVERRIDES_NAMESPACE** 	| `kyma-installer`                      	| Namespace in which the Installer overrides are located 	|
```
